webrtc.html

```diff
@@ -2763,6 +2763,14 @@
               bandwidth.
             </p>
           </dd>
+
+          <dt>unsigned long maxBitrate</dt>
+          <dd>
+            <p>
+              Indicates the maximum bitrate that can used to send this
```

Comment:
Nit: "can be used"

Query: How does it interact with other bitrates (per-transport or per-session), and where should text describing such interaction go?